src/http_microhttpd.c

```diff
@@ -615,19 +627,26 @@ static int encode_and_redirect_to_splashpage(struct MHD_Connection *connection,
  */
 static int redirect_to_splashpage(struct MHD_Connection *connection, t_client *client, const char *host, const char *url)
 {
-	char *originurl = NULL;
+	const char *originurl = NULL;
```

Comment:
please remove the const here, it should not even compile

Reply:
It would not compile without it :-D Will fix.